### PR TITLE
Update players.yaml

### PR DESCRIPTION
**Pull request by gokhantahil**
Instead of Rip, GLX_Rusher has been added. He is in the ladder right now!

### CODE REVIEW

File: data/players.yaml [hunk 1]
```diff
@@ -3561,3 +3561,28 @@
     - '2659351'
     voobly:
     - '124651224'
+- name: Dr.Kozmonot
+  country: tr
+  platforms:
+    de:
+    - '1827866'
+    - '3446882'
```
Comment (simonsan) [inline data/players.yaml:3569]: Not Top200, please remove

**Review by simonsan — CHANGES_REQUESTED** [review]

**Comment by gokhantahil** [comment]
Oh i thought as soon as he participates to tournaments, he could have it. Ok i'll remove.

File: data/players.yaml [hunk 2]
```diff
@@ -3561,3 +3561,22 @@
     - '2659351'
     voobly:
     - '124651224'
+- name: Yasin
+  country: tr
+  platforms:
+    de:
+    - '2944434'
+    - '5829363'
+- name: TurkEycof_AoE
```
Comment (simonsan) [inline data/players.yaml:3570]: Would take the base name here: `TurkEycof` as he is called either `TurkEycof2` or `TurkEycof_AoE`. Probably better to use just the base then.

Comment (gokhantahil) [reply data/players.yaml:3570]: It's not a bad idea, you know how other data is handled. Let's log in as you want. I'll commit right now.

**Review by gokhantahil — COMMENTED** [review]
TurkEycof_AoE changed with TurkEycof

**Review by simonsan — APPROVED** [review]
Looks good to me.

**Comment by gokhantahil** [comment]
Thanks, have fun!

**Comment by happyleavesaoc** [comment]
Merged, thx @gokhantahil @simonsan

**Pull request merged**